### ISSUE TITLE
Preview pane rendering slow, takes 450ms - why?

### ISSUE THREAD

**Issue by bostjan** [roundcube/roundcubemail#5200]
(I am not a frontend expert)

When preview pane is visible and a new message is selected for preview, it takes ~450ms after server response is received to actually render the message.

**Is there a clear reason for that, or is RC just wasting everyone's time?** :)

Procedure to reproduce:
- chrome, open dev tools network tab, disable element inspection selector
- open preview pane
- clear the network tab contents
- click on email to preview
- 500ms delay for doulbeclick detection (separate issue, #5199)
- 100-200ms for server request and response
- then only after another 450ms message is finally rendered (you will se greyed requests for cached preview pane images [buttons, etc])

Alternative procedure (better): Use timeline recording with Chrome. It shows many events, but IANAFEE (IAmNotAFrontEndExpert).

**Comment by alecpl** [comment]
I'm not confirming this. Maybe some plugin (Roundcube or web browser) is involved or it depends on the message content.

**Comment by bostjan** [comment]
Nope, message content is a simple txt of less than 100 byes.
Same result with all plugins disabled.

**Comment by bostjan** [comment]
I did a bit more of testing, and was able to get consistent results, somewhat better:
- i5-4250u @ 1.3GHz ~~ 330ms (mb air + ubuntu 14.04 + chrome)
- i5-5250u @ 1.6GHz ~~ 200ms (intel nuc + ubuntu 16.04 + chrome)

Please close this, until I figure out where the inconsistency between these and original results comes from.

b.
